pandas/tests/series/indexing/test_where.py

```diff
@@ -464,3 +464,34 @@ def test_where_categorical(klass):
     df = klass(["A", "A", "B", "B", "C"], dtype="category")
     res = df.where(df != "C")
     tm.assert_equal(exp, res)
+
+
+@pytest.mark.parametrize("tz", [None, "US/Pacific"])
+def test_where_datetimelike_categorical(tz):
```

Comment:
can you use index_or_series fixture?

Reply:
thatd be awkward since this also does the DatetimeArray and DataFrame cases